recipe/meta.yaml

```diff
@@ -7,32 +8,46 @@ package:
 source:
   url: https://github.com/google/snappy/archive/{{ version }}.tar.gz
   fn: snappy-{{ version }}.tar.gz
-  sha256: 6fa92cde5b2caefd0d9a60336991ba42e5a7ddc3bdc36c5610451373751d0495
+  sha256: {{ sha256 }}
   patches:
     # 1.1.6 crashes because it tries to install README, but the file is now
     # named README.md. This patch is submitted upstream:
     # https://github.com/google/snappy/pull/46
     - fix-autoconf-readme.patch
+    # Have snappy-unittest not run the slow-ish microbenchmarks, but just
+    # the quick correctness tests, without bothering with gflags (which didn't
+    # work out of the box). This is the recommended method in the README.
+    - skip-microbenchmarks.patch
 
 build:
   number: 0
-  skip: true      # [win]
```

Comment:
Also, sorry I forgot this, can we please add the following?

```yaml
  skip: true      # [win and py35]
```

Then re-render. We can do this because Python 3.5 and 3.6 use the same VC version. So we are just picking one of them.